app/scripts/components/common/map/style-generators/raster-timeseries.tsx

```diff
@@ -274,7 +274,7 @@ export function RasterTimeseries(props: RasterTimeseriesProps) {
         /* eslint-enable no-console */
 
         const responseData = await requestQuickCache<any>({
-          url: `${tileApiEndpointToUse}/mosaic/register`,
+          url: `${tileApiEndpointToUse}/searches/register`,
```

Comment:
👋 It will be awesome if we can make this code work regardless of the backend api so we don't have to worry about when the backend code will be shipped. 

For example, do you think it would be possible to check one of the new end point that is included in the new titiler update and let the code to fallback to the old behavior when the new api endpoint doesn't exist? (This is just an idea for an example, it woudl be even better if there is a simpler way to do so!)

Reply:
we could do that but I think we can just feature flag then? Once BE ships to prod, we can turn on flag, if all is good, go in with a follow up to remove flag.

Reply:
But it might be useless to have a fallback because the old endpoints wont work anyways once these changes move to prod on the BE. I think it makes more sense to just have careful coordination with BE? Unless i'm misunderstanding you..

Reply:
I can't think of how to make a feature flag work without stressing over the timing of the backend deployment. Will the flag be set as an environment variable? If so, should the change in this environment variable still be coordinated with the backend deployment timing? I might be missing something, so if you have a concrete idea, I'd love to see it! 

So ideal sequence will be 

- ship dashboard code that works for both old and new endpoints
- release the dashboard with the code from the previous step
- ship backend
- remove the code for old endpoint

Reply:
yep, FF would be instead of checking for a condition, we manually pull a trigger without touching code at point in time but used to doing that with LaunchDarkly which we haven't been doing. Will add this condition so that this change can merge without waiting. We will need the instances to have this change before BE moves to prod 👍🏼 thanks Hanbyul!

Reply:
@hanbyul-here I made this update to fall back back on old urls, can you check to see if that is what you meant? If you know a better way to do this without having to make the request first, please suggest. Also a note.. old and new endpoints will fail with `https://staging-raster.delta-backend.com` anyways because its deprecated